scenarios/hci/roles.yaml

```diff
@@ -0,0 +1,258 @@
+---
+###############################################################################
+# File generated by TripleO
+###############################################################################
+###############################################################################
+# Role: Controller                                                            #
+###############################################################################
+- name: Controller
+  description: |
+    Controller role that has all the controler services loaded and handles
+    Database, Messaging and Network functions.
+  CountDefault: 1
+  tags:
+    - primary
+    - controller
+    # Create external Neutron bridge for SNAT (and floating IPs when using
+    # ML2/OVS without DVR)
+    - external_bridge
+  networks:
```

Comment:
add External here too

Reply:
done, will update the PR once current testing run finishes